package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "hls.js": "^0.14.17",
-    "video.js": "^7.10.2",
+    "video.js": ">=7.12.1",
```

Comment:
I would expect to see the semver version here in the dependency:

- `"^7.12"` << this will include minor releases (`7.12+` and `<8`)
- `"^7.12.1"` << this will include only patch releases (`7.12.1+` and `<7.13`)

I believe we want the former, since I wouldn't expect video.js to be breaking APIs in minor releases.

Reply:
DM'd with Dylan on this and we have a path forward!  Commit coming soon!